docs/getting-started/github-apps/github-apps-troubleshooting-tips.rst

```diff
@@ -152,3 +152,11 @@ automatically detect and sync changes with the corresponding version on Dockstor
 
 If you have a .dockstore.yml file in your master or develop branches on GitHub, any new branches you create from these as your template
 will have a  .dockstore.yml.
+
+
+Why are only some branches appearing on my workflow/tool entry?
```

Comment:
We've discussed this before in other areas, and I know you disagree, but I don't think you should use the word entry here. It should be something like "Why are only some branches appearing on my workflow or tool?".

We generally use entry to mean a tool or a workflow, instead of having to say "tool or workflow". As I understand it, you consider an entry to be the the workflow descriptors plus the other stuff Dockstore adds around it.

We have a slot for a storytime, https://ucsc-cgl.atlassian.net/wiki/spaces/DEV/pages/752779272/Team+Seabright+Storytime, where we can delve into it further (let us know when you're ready!). 

But until then, I think we should stick with an entry being a tool or a workflow, and the title sounds redundant to me.